### ISSUE TITLE
Include external assets in the code base

### ISSUE THREAD

**Issue by m33m33** [syuilo/Misskey#8118]
https://github.com/misskey-dev/misskey/blob/5e4877b8a4edaa89d2550e7c03c861536762ea7a/packages/client/src/ui/chat/notes.vue#L4

There are a few references to external assets like this one. It would be nice to have all graphic assets in the code base, so Misskey instance will be fully autonomous and won't rely on external websites. That, for instance, may be blocked by firewall url filtering, induce latency or connectivity issues.

Plus, you can't customise graphic assets like this if they are hard coded to external ressources.

Current use of external assets:
```
grep -R  "xn--931a.moe" ./*
./README.md:<a href="https://xn--931a.moe/"><img src="https://github.com/misskey-dev/misskey/blob/develop/assets/ai.png?raw=true" align="right" height="320px"/></a>
./packages/client/src/pages/messaging/index.vue:                        <img src="https://xn--931a.moe/assets/info.jpg" class="_ghost"/>
./packages/client/src/pages/not-found.vue:              <img src="https://xn--931a.moe/assets/not-found.jpg" class="_ghost"/>
./packages/client/src/pages/_error_.vue:                <img src="https://xn--931a.moe/assets/error.jpg" class="_ghost"/>
./packages/client/src/pages/settings/apps.vue:                          <img src="https://xn--931a.moe/assets/info.jpg" class="_ghost"/>
./packages/client/src/pages/follow-requests.vue:                                <img src="https://xn--931a.moe/assets/info.jpg" class="_ghost"/>
./packages/client/src/ui/chat/notes.vue:                <img src="https://xn--931a.moe/assets/info.jpg" class="_ghost"/>
./packages/client/src/components/global/error.vue:              <img src="https://xn--931a.moe/assets/error.jpg" class="_ghost"/>
./packages/client/src/components/notes.vue:             <img src="https://xn--931a.moe/assets/info.jpg" class="_ghost"/>
./packages/client/src/components/ui/pagination.vue:                             <img src="https://xn--931a.moe/assets/info.jpg" class="_ghost"/>
./packages/backend/migration/1000000000000-Init.js:        await queryRunner.query(`CREATE TABLE "meta" ("id" character varying(32) NOT NULL, "name" character varying(128), "description" character varying(1024), "maintainerName" character varying(128), "maintainerEmail" character varying(128), "announcements" jsonb NOT NULL DEFAULT '[]', "disableRegistration" boolean NOT NULL DEFAULT false, "disableLocalTimeline" boolean NOT NULL DEFAULT false, "disableGlobalTimeline" boolean NOT NULL DEFAULT false, "enableEmojiReaction" boolean NOT NULL DEFAULT true, "useStarForReactionFallback" boolean NOT NULL DEFAULT false, "langs" character varying(64) array NOT NULL DEFAULT '{}'::varchar[], "hiddenTags" character varying(256) array NOT NULL DEFAULT '{}'::varchar[], "blockedHosts" character varying(256) array NOT NULL DEFAULT '{}'::varchar[], "mascotImageUrl" character varying(512) DEFAULT '/assets/ai.png', "bannerUrl" character varying(512), "errorImageUrl" character varying(512) DEFAULT 'https://xn--931a.moe/aiart/yubitun.png', "iconUrl" character varying(512), "cacheRemoteFiles" boolean NOT NULL DEFAULT true, "proxyAccount" character varying(128), "enableRecaptcha" boolean NOT NULL DEFAULT false, "recaptchaSiteKey" character varying(64), "recaptchaSecretKey" character varying(64), "localDriveCapacityMb" integer NOT NULL DEFAULT 1024, "remoteDriveCapacityMb" integer NOT NULL DEFAULT 32, "maxNoteTextLength" integer NOT NULL DEFAULT 500, "summalyProxy" character varying(128), "enableEmail" boolean NOT NULL DEFAULT false, "email" character varying(128), "smtpSecure" boolean NOT NULL DEFAULT false, "smtpHost" character varying(128), "smtpPort" integer, "smtpUser" character varying(128), "smtpPass" character varying(128), "enableServiceWorker" boolean NOT NULL DEFAULT false, "swPublicKey" character varying(128), "swPrivateKey" character varying(128), "enableTwitterIntegration" boolean NOT NULL DEFAULT false, "twitterConsumerKey" character varying(128), "twitterConsumerSecret" character varying(128), "enableGithubIntegration" boolean NOT NULL DEFAULT false, "githubClientId" character varying(128), "githubClientSecret" character varying(128), "enableDiscordIntegration" boolean NOT NULL DEFAULT false, "discordClientId" character varying(128), "discordClientSecret" character varying(128), CONSTRAINT "PK_c4c17a6c2bd7651338b60fc590b" PRIMARY KEY ("id"))`);
./packages/backend/src/server/web/views/base.pug:               link(rel='prefetch' href='https://xn--931a.moe/assets/info.jpg')
./packages/backend/src/server/web/views/base.pug:               link(rel='prefetch' href='https://xn--931a.moe/assets/not-found.jpg')
./packages/backend/src/server/web/views/base.pug:               link(rel='prefetch' href='https://xn--931a.moe/assets/error.jpg')
./packages/backend/src/server/api/endpoints/meta.ts:                            default: 'https://xn--931a.moe/aiart/yubitun.png',
./packages/backend/src/models/entities/meta.ts:         default: 'https://xn--931a.moe/aiart/yubitun.png',
```

**Comment by Johann150** [comment]
Duplicate of #7043
